microsoft-edge/toc.yml

```
@@ -1362,6 +1362,10 @@
             - name: Automate and test with Microsoft Edge WebDriver
               href: webview2/how-to/webdriver.md
               displayName: Automate and test WebView2 apps with Microsoft Edge WebDriver  # top-of-page title
+            
+            - name: Data and Privacy
```

```suggestion
            - name: Data and privacy
```
per corp style guides, use sentence case for titles/headings.  Unless you want "Privacy" as a special exception like What's New or Release Notes.